include/ck/wrapper/layout.hpp

```diff
@@ -43,11 +49,18 @@ struct Layout
             Number<Tuple<Ts...>::Size()>{});
     }
 
-    // Generate LowerDims in Compile-time for MergeTrasform using passed Type
-    // If element of Tuple<Ts...> is also tuple, then merge (generate sequence for merge)
-    // If tuple is element, then pass through (sequence with one element)
+    /**
+     * \brief Generate LowerDims in Compile-time for MergeTrasform using
+     * passed Type. If element of Tuple<Ts...> is also tuple, then merge
+     * (generate sequence for merge). If tuple is element, then pass through
```

Comment:
just from reading a description I don't get it how a tuple can be an element? Could you clarify it a bit more?